openfga/v2beta1/openfga.proto

```diff
@@ -0,0 +1,163 @@
+syntax = "proto3";
+
+package openfga.v2beta1;
+
+import "google/protobuf/struct.proto";
+
+message RelationshipTuple {
```

Comment:
WHat do you think about naming them `RelationshipTuple` and `RelationshipTupleWithCondition`?

Reply:
So invert them?

I think the API more unanymously operates on a RelationshipTuple which can optionally have a condition. For example the `WriteRelationshipTuples` API, the `ReadRelationshipTuples` API, etc.. The exception are those APIs that do not need a condition in the tuple representation whatsoever. So I think I lean toward leaving as is.